### ISSUE TITLE
Live editing

### ISSUE THREAD

**Issue by daralthus** [Jam3/devtool#7]
Anyone managed to get live modifications to the files in the sources panel work?

**Comment by mattdesl** [comment]
I was looking at this today.

It can work if I add the current working directory to the DevTools workspace. However, when the debugger hits, it opens a different file than the one that has local modifications. Saving the newly opened tab errors out (everything is tinted red). Even though both of these tabs point to the same link, there is something not working on the Chrome/Electron end. I may need to set some path somewhere... Not sure.

For what it's worth, it seems like `iron-node` has the same problem.

**Comment by mattdesl** [comment]
Tracking upstream – we are blocked on this: https://github.com/atom/electron/issues/2322

**Comment by mattdesl** [comment]
Also see here:
https://github.com/atom/electron/issues/4185

It seems like this is a "wontfix" in Electron. I will close this issue until Electron exposes this feature. If anybody has another way around it, please let me know. :smile:
